customers/schema/mutations.py

```diff
@@ -218,6 +220,38 @@ def mutate_and_get_payload(cls, root, info, **input):
         return CreateBerthServicesProfileMutation(profile=profile)
 
 
+class CreateMyBerthProfileMutation(graphene.ClientIDMutation):
+    class Input:
+        profile_token = graphene.String(
+            required=True, description="API token for Helsinki profile GraphQL API",
+        )
+
+    profile = graphene.Field(ProfileNode)
+
+    @classmethod
+    @login_required
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, profile_token, **input):
+        user = info.context.user
+        profile_exists = CustomerProfile.objects.filter(user=user).exists()
+        if profile_exists:
+            return None
+
+        profile_service = ProfileService(profile_token=profile_token)
+        my_profile = profile_service.get_my_profile()
+        if not my_profile:
+            raise VenepaikkaGraphQLError("Open city profile not found")
+
+        try:
+            profile = CustomerProfile.objects.create(id=my_profile.id, user=user)
```

Comment:
Does the login/sign in process actually create a user on the system? Since we're not using `user` for anything, idk how valuable is it to keep store it.

Reply:
`GraphQLApiTokenAuthentication` will create an `User` instance if it does not exist using `helusers.oidc.ApiTokenAuthentication.authenticate` -> `helusers.oidc.resolve_user` -> `helusers.user_utils.get_or_create_user`

The main reason I added it here so that we can check if the `CustomerProfile` exists for the user without querying the profile backend:
```
profile_exists = CustomerProfile.objects.filter(user=user).exists()
```

Reply:
Can't we replace the `user resolver` with our own that would fetch the `CustomerProfile` based on the profile id on the JWT claims? I think start using `user` now would add another layer of unnecessary complexity.

Reply:
On second though... since now the customers become _users_ of our system, it would kinda make sense that we add them as users.

I was thinking that, if we replace the resolver, then the admins who don't have a customer profile won't be able to access the system anymore (since we'd be looking for CP, not for Users).

Reply:
We _mabye_ should add another permission `Group` customer in case it'd be necessary and assign it to this users by default..? Not very sure about this one.

Reply:
I agree that the concept of `User` makes good sense for customers.
Adding a "customers" `Group` sounds good to me.